server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java

```diff
@@ -423,13 +423,14 @@ public void onNewInfo(ClusterInfo info) {
 
     // exposed for tests to override
     long sizeOfRelocatingShards(RoutingNode routingNode, DiskUsage diskUsage, ClusterInfo info, ClusterState reroutedClusterState) {
-        return DiskThresholdDecider.sizeOfRelocatingShards(
+        return DiskThresholdDecider.sizeOfUnaccountedShards(
```

Comment:
I'm not sure if it is correct to use 0L here? this might mess up the disk thresholds monitoring?

Reply:
Don't have access to `RoutingAllocation` here, so I'm not sure what's a better approach here.